django/utils/deconstruct.py

```diff
@@ -23,7 +23,7 @@ def deconstruct(obj):
             and keyword arguments.
             """
             # Fallback version
-            if path:
+            if path and obj.__class__.__name__ in path:
```

Comment:
why not just `type(obj) is klass` ?

Reply:
Yeah that's a simpler alternative, will do it.